src/lib/style/base/_base.scss

```diff
@@ -21,12 +21,9 @@ html {
   box-sizing: inherit;
 }
 
-body * {
-  font-family: 'Lato';
-}
-
 body {
   color: $gray-regular;
+  font-family: 'Lato';
 }
 
```

Comment:
This is the main fix.  "body *" is a bit too aggressive a selector, it applied to <pre><span class="token string/operator/punctuation/..."></span></pre>, and changed the font family in codemirror.  The way codemirror works (today) is that it has a textarea (with absolute position) overlaying a pre, where the pre is used to show the actual code (with syntax highlighting, etc).  Once we mucked with the style, these two were no longer in sync, and the cursor was no longer on the textfield text that the user wants to edit!